site/public/css/server.css

```diff
@@ -393,7 +393,17 @@ table tr.table-header {
     font-size: 20px;
 }
 
-#header-text>h2:nth-child(2) {
+#header-text>div {
+    display: block;
+    font-size: 1.5em;
+    margin-block-start: 0.83em;
+    margin-block-end: 0.83em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+    font-weight: bold;
+}
+
+#header-text>div:nth-child(2) {
```

Comment:
I don't think this rule gets triggered anymore and could be deleted.

Reply:
Done.